packages/medusa/src/models/cart.ts

```diff
@@ -124,6 +124,8 @@ export class Cart {
   @PrimaryColumn()
   id: string
 
+  readonly object = "cart"
```

Comment:
More of a question here rather than a comment: what's the idea behind this field? I suppose it's because at some point, we want to discriminate or check at runtime if some `x` is a `Cart` right? If yes, does something like `x instanceof Cart` not work?